x-pack/test/detection_engine_api_integration/security_and_spaces/group10/risk_engine/utils.ts

```diff
@@ -167,6 +164,16 @@ export const legacyTransformIds = [
   'ml_userriskscore_latest_transform_default',
 ];
 
+export const clearTransforms = async ({ es }: { es: Client }): Promise<void> => {
+  try {
+    await es.transform.deleteTransform({
+      transform_id: 'risk_score_latest_transform_default',
+    });
+  } catch (e) {
+    //
```

Comment:
Maybe either log something here, or make a not that we don't care if this failed (and why)?